src/js/lib/common/svg-export.js

```diff
@@ -1,30 +1,33 @@
-var SVG = SVG || {};
+define([], function() {
```

Comment:
I'm starting to think these 3 files belong in js/angular, not in js/lib

Reply:
yeah makes sense I can move them in a `js/angular/common/` library. @desislava-hristova-ontotext what do you think?

Reply:
It's not obvious, but these are third party libraries. See https://docs.google.com/spreadsheets/d/12cIovBIlN_xZph3P6DuFn4dvqNSR6F0ctqFy5k0n41g/edit#gid=0
I prefer keeping them in lib for now.

Reply:
ah I see. We though that these are ours :)
I'll move them back in lib then